test/test_fapi.py

```diff
@@ -72,8 +72,7 @@ class TestFapi:
     @pytest.fixture
     def esys(self):
         # TODO ESAPI should accept either a cdata obj or a dedicated TCTI class, not TctiLdr!
-        tcti = SimpleNamespace()
-        tcti.ctx = self.fapi.tcti
+        tcti = TCTI(self.fapi.tcti)
```

Comment:
shouldn't FAPI's property function `tcti` return the TCTI class?

Reply:
Yes. The TCTI should make the underlying `cdata` object available anyway.